resources/tests/genCerts.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+: ${COUNTRY:=US}
+: ${STATE:=CO}
+: ${LOCALE:=FortCollins}
+: ${COMPANY:=Vertica}
+: ${ORG:=SQL}
+: ${CN:=localhost}
+
+: ${CERT_LOC:=./resources/tests/ssl}￼
```

Comment:
What is this ￼?

Reply:
I'm sorry, I don't understand the question. Github shows me a box with OBJ in it before the question mark.
Are you asking about the value of these fields, or the construct to get their values, or...

Reply:
Github shows me a box with OBJ in it. Is this a meaningful value or a typo?

Reply:
That's the only place I saw this symbol. It is visible in Sublime with UTF-8 encoding from my end. Probably this is a matter of the setting of your editor.